common/app/views/fragments/analytics/google.scala.html

```diff
@@ -13,9 +13,11 @@
     var pillar = '@{navMenu.currentPillar.map(_.title).getOrElse("")}';
 }
 
+// This is to prevent console errors from code calling window.ga for sessions where we have not loaded the tracking script
+// (where the user has not given their consent for GA tracking)
+window.ga = function() {};
 
-(function() {
-
+window.guardian.config.googleAnalytics.initialiseGa = function() {
```

Comment:
I'm not sure this should live on 'window.guardian.config' but I'm also wary of defining something else on [window.guardian](https://github.com/guardian/frontend/blob/main/common/app/templates/inlineJS/blocking/config.scala.js#L20) and there is already something called 'window.guardian.config.googleAnalytics' which seems like an okay place?

Reply:
To me, it's not really config - I'd keep it on `window.guardian` personally

Reply:
Done, thanks Jon